dev_scripts/find_images_to_rebuild.py

```diff
@@ -40,15 +46,33 @@
 
 errata = Errata()
 kwargs = {}
-if container_images:
+
+if args.container_img:
     EventClass = ManualRebuildWithAdvisoryEvent
-    kwargs['container_images'] = container_images
+    kwargs['container_images'] = args.container_img
 else:
     EventClass = ErrataAdvisoryStateChangedEvent
+
+event_id = None
+if args.cassette:
+    cassette_name = os.path.splitext(os.path.basename(args.cassette))[0]
+    extension = os.path.splitext(os.path.basename(args.cassette))[1]
+    try:
+        event_id = int(cassette_name)
```

Comment:
> @mulaievaRH just one small comment to address. Were you able to test this to see if it works?

@mprahl , Yes, I've tested it. For the test I've deleted here int() for trying `--cassette-path ~/vcr_test/fixtures/cassettes/fake_message.yml` (because I have not a 'real' cassette ). It works.